development-defaults.ini

```diff
@@ -84,12 +84,23 @@ orange16 = "Orange 16"
 orange17 = "Orange 17"
 orange18 = "Orange 18"
 orange19 = "Orange 19"
-cabin01 = "Cabin 01 - GoldFinch - Music"
-cabin02 = "Cabin 02 - Ibis - Music"
-cabin03 = "Cabin 03 - Falcon - Music"
-cabin04 = "Cabin 04 - Blue jay (lot blue 9)"
-cabin05 = "Cabin 05 - Junco (Lot Yellow 25)"
-manor_house01 = "Manor House"
+"dwelling01 = "Manor House"
```

Comment:
The double-quotes in front of these variables will break the system if we try to deploy this. They all need to be removed.